src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/Utils.cs

```diff
@@ -754,7 +754,11 @@ internal static bool IsSelfSigned(X509Chain chain)
 
         internal static AsymmetricAlgorithm GetAnyPublicKey(X509Certificate2 certificate)
         {
-            return (AsymmetricAlgorithm)certificate.GetRSAPublicKey() ?? certificate.GetDSAPublicKey();
+            return (AsymmetricAlgorithm)certificate.GetRSAPublicKey()
+#if netcoreapp || uap || uapaot
```

Comment:
I don't think anyone would notice or care if DSA support was missing.  That said, I don't know why we'd add this as netstandard instead of just netcoreapp; as a netstandard build it would need to do a netfx build forwarding back to System.Security.dll.